content/_data/roadmap/roadmap.yml

```
@@ -689,6 +689,15 @@ categories:
       - documentation
       - community
       - events
+    - name: Hacktoberfest 2021
+      status: present
```

```suggestion
      status: current
```
As per https://github.com/jenkins-infra/jenkins.io/pull/4618/files#diff-ce5dcb079a0375d6bd9b077279a7752d40aa5479201ff41abf018a6907e360acR649